scripts/golangci.yml.in

```diff
@@ -165,3 +165,5 @@ linters:
     - wsl
     - funlen
     - testpackage
+    - goerr113
+    - nestif
```

Comment:
if you disable `nestif` explain why that is needed (each linter disabling its own commit) and add a reference to an issue

Reply:
Okay, will do that.